p2pv2/pb/util.go

```diff
@@ -39,6 +41,35 @@ func CalDataCheckSum(msg *XuperMessage) uint32 {
 	return crc32.ChecksumIEEE(msg.GetData().GetMsgInfo())
 }
 
+// Compressed compress msg
+func Compressed(msg *XuperMessage) *XuperMessage {
```

Comment:
Compress or Compressed?

Reply:
> Compress or Compressed?

It's a better way to use compress. Thank you for your advice.